src/PyCall.jl

```diff
@@ -298,6 +300,14 @@ function setindex!(o::PyObject, v, s::Union{Symbol,AbstractString})
     o
 end
 
+getproperty(o::PyObject, s::Symbol) = getindex(o, s)
+getproperty(o::PyObject, s::AbstractString) = getindex(o, s)
+
+propertynames(o::PyObject) = map(x->Symbol(first(x)), 
+                                pycall(inspect["getmembers"], PyObject, o))
+
+setproperty!(o::PyObject, s::Union{Symbol,AbstractString}, v) = setindex!(o, v, s)
```

Comment:
similarly, move the definition of `setindex!` here.

Reply:
Done.